.githooks/README.md

```diff
@@ -0,0 +1,28 @@
+# .githooks
+
+This folder contains executable files that will be invoked by git at certain git operations.
+
+By default git hooks are located i `.git/hooks` folder at the root of the repository. Since the default folder is hidden by most IDEs, this repository reconfigures git's hook location in order to make the hooks visible and easier to maintain.
+
+## Configuration
+
+Normal development flows (see file [../README.md](../README.md) ) will call git to change the location of git hooks.
```

Comment:
Does this file have any reference to anything related here?

Reply:
This was mainly for people to understand where the development setup is documented, nothing specific to this change.